app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/name.phtml

```diff
@@ -4,33 +4,36 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block */
+/**
+ * @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```

Comment:
Please remove not needed closing and opening PHP tags and combine all PHP dock blocks

Reply:
fixed